workspaces/cli-shared/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "outDir": "build",
     "rootDir": "src",
+    "strict": false
```

Comment:
Changes like this should be discussed to make sure they are necessary since they e.g. allow for more failure modes. Why is this necessary?